stories/Molecules/Text/Faq/Faq.scss

```diff
@@ -9,40 +9,53 @@
 }
 
 .accordion__item {
-    border-bottom: 1px solid $color-gray-500;
-    position: relative;
-    padding: $spacing-16 0 $spacing-24;
-    cursor: pointer;
-    overflow: hidden;
-    list-style: none;
+  border-bottom: 1px solid $color-gray-500;
+  position: relative;
+  padding: $spacing-16 0 $spacing-24;
+  cursor: pointer;
+  overflow: hidden;
+  list-style: none;
+  &::before {
+    content: "";
+    background: url(#{$img-path-Icon}/Chevron-down.svg) no-repeat center center;
+    float: right;
+    width: 20px;
+    height: 20px;
+    border-top-color: #f3f3f3;
+    transition: all 0.1s ease;
```

Comment:
@harishkumar-srijan can you please use the mixin for the transition created by pallavi

Reply:
Done